packages/docs/docs/plugin/option-api.md

```diff
@@ -434,3 +434,29 @@ Then, VuePress will automatically inject these components behind the layout comp
 </div>
 </div>
 ```
+
+## registerCommand
+
+- Type: `function`
+- Default: `undefined`
+
+Register a extra command to enhance the CLI of vuepress. The function will be called with a [CAC](https://github.com/cacjs/cac)'s instance as the first argument.
+
+```js
+module.exports = {
+  registerCommand (cli) {
+    cli
```

Comment:
why call it `registerCommand` when you can get the entire `cli` instance 😂

Reply:
How is `extendCLI`?

Reply:
Maybe passing a proxied CLI instance would be better, for now we should only have the ability ti extend the command.

Reply:
Renamed to `extendCli` at https://github.com/vuejs/vuepress/pull/1069/commits/1f7bf85bdd7e96e4f759d3785c96b60e32e5fa83